tests/test_application_form.py

```diff
@@ -106,8 +106,8 @@ def test_form_pages_accessible(self):
                     },
                     route_rel,
                 )
-                assert len(results["violations"]) <= 2
-                assert len(results["violations"]) == 0 or all(
+                assert len(results["violations"]) <= 4
+                assert len(results["violations"]) <= 4 or all(
                     [
```

Comment:
This will allow four serious/critical accessibility violations. The causes of the serious/critical violations should really be fixed where they're occurring rather than allowing them through here.

Reply:
This pull request is uncontroversial but nevertheless it raises some questions about how we've handled a11y violations - I think a test config should be created and used to track anywhere that violations are allowed because of third party libraries. We can also create an issue in the x-gov-form-builder which states where is a serious a11y violation in the summary page.

Reply:
The accessibility issue is coming from the summary page. I  had communicated with Jen for further advice & she has advised raising a ticket so she can work on it.

Reply:
I have  logged the issue on Github: https://github.com/XGovFormBuilder/digital-form-builder/issues/765 & 

raised a ticket on Jira: https://digital.dclg.gov.uk/jira/browse/FS-629

Reply:
Great Work Ram! Very detailed Jira ticket, I will add to the X-Gov-Form ticket on where to fix the issue!

Reply:
Thank you, Jack

Reply:
Great tickets/issues Ram. This line still allows four serious accessibility violations **per page** though. Is this what we want to do? If needed for now to finish this sprint, perhaps we should add a ticket in Jira to note the fact that the accessibility tests on this main repo currently allow (up to 4) serious accessibility violations per page? (So we don't forget to bring the threshold down later?)

Reply:
Thanks, Harry, with this we can see the summary page & the submit button & if I remove the summary page, it means the submit button would be removed however the application would still be submitted but from the user's point of view, I think the submit & summary page is important? I have already raised the ticket on Jira & logged the issue on X-Gov GitHub & communicated with Jen. I will keep chasing this issue with Jen.

Reply:
Thanks Harry, I have raised the ticket on Jira & logged an issue on X-Gov Github, communicated with Jen & I will keep chasing with Jen

Reply:
I have raised an issue on Github with ref: https://github.com/communitiesuk/funding-service-design-frontend/issues/34

Also, I have raised a ticket on Jira: https://digital.dclg.gov.uk/jira/secure/RapidBoard.jspa?rapidView=110&view=planning&selectedIssue=FS-636&issueLimit=100

Reply:
Nice work Ram